packages/ens-sync-script/src/index.ts

```diff
@@ -86,6 +87,11 @@ async function main() {
     // streamRegistryContract.on("StreamCreated", async (streamId, metadataJsonString) => {
     //     log("Got StreamCreated event params: ", streamId, metadataJsonString)
     // })
+
+    // send heartbeat every 10 minutes
+    setInterval(() => {
+        fs.writeFileSync("heartbeat", "")
+    }, 1000 * 10)
```

Comment:
Every 2 minutes would still be fine and not kill peoples systems.

Also let's add an initial heartbeat so `streamr-docker-dev --wait` can proceed in CI runs.

```suggestion
    // initial heartbeat (5 seconds safety margin to wait for contract listener to be active)
    setTimeout(() => {
    }, 5 * 1000)
    // thereafter send heartbeat every 2 minutes
    setInterval(() => {
        fs.writeFileSync("heartbeat", "")
    }, 2 * 60 * 1000)
```